bluemira/builders/EUDEMO/plasma.py

```diff
@@ -165,7 +162,6 @@ def read(self):
         eq = self._read_equilibrium()
         self._analyse_equilibrium(eq)
         self._boundary = make_polygon(eq.get_LCFS().xyz, "LCFS")
-        return {"equilibrium": eq}
```

Comment:
Shouldn't we set `self._equilibrium` here?

Reply:
It's set in `_analyse_equilibrium` that way whenever the parameters are updated the equilibrium gets updated too... coming to think of it, the same logic also follows for updating the boundary in `_analyse_equilibrium` to ensure that is kept aligned too... I've added that in too.